siteapp/views.py

```diff
@@ -1418,6 +1418,59 @@ def new_portfolio(request):
         "project_form": ProjectForm(request.user),
     })
 
+@login_required
+def delete_portfolio(request, pk):
+    """Form to delete portfolios"""
+    if request.method == 'GET':
+        portfolio = Portfolio.objects.get(pk=pk)
+        Portfolio.objects.get(pk=pk).delete()
+
+        logger.info(
+            event="delete_portfolio",
+            object={"object": "portfolio", "id": portfolio.id, "title": portfolio.title},
```

Comment:
Another minor nit-pick: This is technically logging before the delete happens. If there is an error with the delete (such as the portfolio not being found) the log will still happen, which might be confusing. I don't think it's likely, since the portfolio is retrieved just 1 step prior, but something to consider. I was initially thinking to store the portfolio _properties_ in variables before the deletion, but not a huge deal.

Reply:
I moved the logging up so this should be resolved